app/models/conference_preference.rb

```diff
@@ -7,10 +7,18 @@ class ConferencePreference < ActiveRecord::Base
   belongs_to :first_conference, class_name: 'Conference'
   belongs_to :second_conference, class_name: 'Conference'
 
+  scope :current_teams, -> {
```

Comment:
It would be cool, if you could add a test for this scope in the `ConferencePreference` model specs.